npc/quests/skills/merchant_skills.txt

```diff
@@ -41,7 +41,7 @@ alberta,83,96,5	script	Necko	98,7,7,{
 			mes "Ppyakikakikakakakakakakaka!!";
 			close;
 		}
-		else if (JobLevel < 15) {
+		else if (JobLevel < 15 && (BaseJob != Job_Blacksmith || BaseJob != Job_Alchemist ) {
```

Comment:
In part you're right. According to our Aegis script the following dialogue should appear for normal job merchant job level below 15 only.
`else if (JobLevel < 15 && Class == Job_Merchant) {`